src/models/mods.rs

```diff
@@ -72,14 +73,16 @@ pub struct Version {
     /// A list of files available for download for this version.
     pub files: Vec<VersionFile>,
     /// A list of mods that this version depends on.
-    pub dependencies: Vec<ModId>,
-    /// A list of versions of Minecraft that this version of the mod supports.
-    pub game_versions: Vec<GameVersion>,
+    pub dependencies: Vec<VersionId>,
+    /// The loaders that this version works on
+    pub loaders: Vec<String>,
```

Comment:
We should keep this field, but we should probably also add a list of the supported loaders to each `VersionFile`, as some mods have multiple files for each version, each supporting a certain mod loader.

Also for mod loader compatibility, it may be a good idea to specify the dependencies down to a specific file, in case the mod loaders don't match.

Reply:
I re-added the field. I think it's a little too complex for modders if every single file had a loader. If it's a forge version, it should probably not be with the fabric version as that causes confusion. It would also make it harder to present downloads on the site.

Reply:
The reason why loaders is a vector is that some mods have the forge version and fabric version bundled.